src/test/java/com/wikia/webdriver/common/remote/discussions/DiscussionsOperations.java

```diff
@@ -115,4 +125,20 @@ public void validatePost(PostEntity.Data data) {
   public void validatePost(ModeratePostContext context) {
     new ValidatePost(user).execute(context);
   }
+
+  private void upvote(UpvoteContext context) {
+    new UpvotePost(user).execute(context);
+  }
+
+  private void upvote(String siteId, String postId) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "upvote" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)